lua/autorun/server/sv_autoclean.lua

```diff
@@ -9,8 +9,8 @@ local clientCleanupCommands = {
     ["stopsound"] = true
 }
 
-local function notifyPlayers( notification )
-    local message = "[CFC_Autoclean] " .. notification
+local function notifyPlayers( notificationMsg )
+    local message = notificationMsg
```

Comment:
i think we should just remove the local variable and change the parameter name to `message`

Reply:
I can get behind that